src/mgos_ili9341_font.c

```diff
@@ -167,9 +167,133 @@ bool mgos_ili9341_set_font(GFXfont *f) {
     s_font = f;
     return false;
   }
-  if (f->font_width==0 && f->font_height==0)
+  if (f->font_width == 0 && f->font_height == 0)
     ili9341_analyzeFont(f);
   s_font = f;
   s_font_type = GFXFONT_INTERNAL;
   return true;
 }
+
+#include "fonts/FreeMono12pt7b.h"
```

Comment:
In the current form, fonts are included by the application and not the library, so that we avoid compiling in and using flash space unnecessarily. By adding these includes here, all fonts will always be linked in (and they're quite a few KB of flash consumption each, so it can add up). I'd prefer it if you did not do that.

Reply:
Suggestion to do this:
1) compile the library as-is (with all fonts included), and make note of the resulting app or .a size (A)
1) compile the library when adding exactly one font (say `FreeMono12pt7b.h` here and in `static const struct font_map fonts[]` below, removing the others), make note of resulting app or .a size (B)

Can you respond with (A) and (B) numbers so we can make a call if the addition is ~acceptable?

Reply:
app.bin sizes:
A - esp8266 - 975360/esp32 - 1155856
B - esp8266 - 757472/esp32 - 936656
A-B - esp8266 - 217888/esp32 - 219200